### ISSUE TITLE
Turn off microservices

### ISSUE THREAD

**Issue by Medic97** [imagegenius/docker-immich#35]
I am trying to upload 60k worth of photos and the machine learning is taking up useful power away from the upload to generate thumbnails and stuff. I wanted to pause all jobs while the upload is happening and it was recommended I turn off the microservices container. 

Is there a way in the AIO to do that? Maybe through CLI?

**Comment by Medic97** [comment]
I ran the check function first which came back with the "up (pid 1949) 73937 seconds". 
I then ran the first one which came back with this "OCI runtime exec failed: exec failed: unable to start container process: exec: "svc": executable file not found in $PATH: unknown"

I'm running in docker on my QNAP if that matters.

**Comment by hydazz** [comment]
i have fixed the command (missed the `s6`)

**Comment by Medic97** [comment]
Just seems to hang and do nothing.

**Comment by hydazz** [comment]
are you using `docker exec ... s6-svc -wD /run/service/svc-microservices` or `docker exec -it ... /bin/bash` then putting in the commands

**Comment by Medic97** [comment]
Sorry complete noob here setup docker for the first time 2 days ago. I was just pasting in the commands as given in the execute console which I assumed ran the command in the container itself. I went there since when I did it through SSH it didn't recognize the command. The check command worked fine as is so I tried the others.

<img width="753" alt="Screenshot 2023-03-20 at 6 20 22 PM" src="https://user-images.githubusercontent.com/23649478/226478133-d7dbd072-091b-4b31-9539-705e8aced975.png">
